src/shared/api/CivicAPI.ts

```diff
@@ -0,0 +1,265 @@
+import $ from 'jquery';
```

Comment:
Also, do we really need `jQuery` here? Maybe we can use `superagent` instead? Or even better, we can auto-generate the client API if there is a swagger definition for that.